### DISCUSSION TITLE
Consulta segunda entrega TPA

### DISCUSSION THREAD

**Discussion by LaSudestada** [dds-jv/foro#422] [TPA - 2024] [answered]
Buenos días/tardes profesor
Tengo una inquietud respecto a la segunda entrega del TPA, según el cronograma se entrega el viernes 24, pero no encuentro la consigna, no se si me estoy perdiendo de algo o si no estoy viendo el link, tambien me preocupa que mis compañeros no digan nada, como si ellos no estuvieran teniendo ningún problema
Si me puede orientar se lo agradeceré mucho
Saludos!

**Comment by flbulgarelli** [comment, marked as answer]
¡Hola! 

Es correcto, aún no publicamos la entrega porque le estamos haciendo algunos ajustes. Vamos a retrasar la entrega una semana.

Saludos
